cloudinary-core/src/main/java/com/cloudinary/Url.java

```diff
@@ -408,6 +409,7 @@ public String generate(String source) {
         String join = StringUtils.join(new String[]{prefix, finalResourceType, signature, transformationStr, version, source}, "/");
         String url = StringUtils.mergeSlashesInUrl(join);
 
+
```

Comment:
redundant line?

Reply:
Not as far as I know (this was not added by me)
join = http://test123-res.cloudinary.com/image/authenticated////test
url = http://test123-res.cloudinary.com/image/authenticated/test
So I think both functions are required. 
Not sure yet why join has all those extra '/'

Reply:
I think he was talking about line 412, that's empty :)

Reply:
Fixing